packages/http/test/index.js

```diff
@@ -769,15 +763,13 @@ describe('tls', () => {
       configuration: {
         label: 'my custom SSL cert',
         publicKey: 'something@mamadou.org',
-        privateKey: 'abc123',
+        ca: 'abc123',
```

Comment:
This test still passes if I comment out `ca`. Actually I can comment out the whole `configuration` object and it'll still pass.

So what is it testing? What's its value? Why do we have this test if it isn't checking the cert?